lsp-dockerfile.el

```diff
@@ -36,20 +36,29 @@
   :link '(url-link "https://github.com/rcjsuen/dockerfile-language-server-nodejs"))
 
 (defcustom lsp-dockerfile-language-server-command
-  '("docker-langserver" "--stdio")
+  '(nil "--stdio")
```

Comment:
this being nil is hard to explain. Can you keep the previous value and change `(cl-first lsp-dockerfile-language-server-command)` to `(executable-find (cl-first lsp-dockerfile-language-server-command))` in the connection function?

Reply:
Done